### ISSUE TITLE
Rework the MetaTask of Observing the Night Sky

### ISSUE THREAD

**Issue by mokun** [mars-sim/mars-sim#1209]
**Is your feature request related to a problem? Please describe.**
- Currently, `ObserveAstronomicalObjectsMeta` implements `SettlementTask`.
- It does not extends `FactoryMetaTask`.
- A person cannot initiate this task by oneself. 

**Describe the rationale**
- Observing the night sky should be a fun activity for anyone 
- It should not be limited to whether he's writing a scientific paper on this topic or whether he's an astronomer or has a science role, etc.

**Describe the goal(s) you have in mind**
- Make it a group activity for everyone to join in.
- Someone may take the lead role to operate the telescope. Others may simply chime in the sky show.
- Allow it to be selectable in Command Dashboard's Logistic Tab's Task Order combo box.
  -- Allow all other `SettlementTask` to be selectable in Command Dashboard

**Screenshots**
- Ryan is an astronomer
![image](https://github.com/mars-sim/mars-sim/assets/1168584/643c09c6-55da-48be-b4a1-4b8712fdcc68)
- The Command Dashboard's Logistic tab's Task Order Combobox does not have any `SettlementTask` to choose from
![image](https://github.com/mars-sim/mars-sim/assets/1168584/e7542df7-4c8e-441f-b16d-c1701d7ada2c)

**Specifications**
- 3.7.0 build 9207

**Additional context**
- Q: Should we have two task, one for professional astronomers doing research using the telescope, the other for amateurs having fun using the telescope's non-research hours ?

**Comment by mokun** [comment]
@bevans2000,

Interestingly, once in a while, we may find settlers going to the observatory for tasks other than observing the night sky. If that's the case we tend to happen, why NOT having fun observing the night sky ? Is astronomer needed as an instructor or anyone having a telescope operator certification ?

For instance, Ivann prefers to go there for reading. See below : 
![image](https://github.com/mars-sim/mars-sim/assets/1168584/e8362256-784c-4647-919d-acf1ca2effbe)

**Comment by bevans2000** [comment]
I guess that is ok. We probably need to weight the Building selection in the general Tasks so they favour buildings in the same zone

**Comment by bevans2000** [comment]
So I think this is a Settlement task because it is linked to a scientific study which is a Settlement level entity. 
People doing it for fun would be a different MetaTask. But would anyone really have time to eva to the observatory for fun?

**Comment by mokun** [comment]
Will you be resuming the *Project* concept for 3.8.0 ? 

I see the star party idea can be the very first indoor group activity under the Project architecture.

**Comment by bevans2000** [comment]
I am.planning in working through the issues assigned to me that are 3.8

**Comment by mokun** [comment]
@bevans2000 

This rework should nicely fit into your idea of the [Group Activity](https://github.com/mars-sim/mars-sim/pull/1281) for those who want to go to a star party in the Observatory.

**Comment by bevans2000** [comment]
> @bevans2000
> 
> This rework should nicely fit into your idea of the [Group Activity](https://github.com/mars-sim/mars-sim/pull/1281) for those who want to go to a star party in the Observatory.

Yes good idea. Could have a social event starting at 800 mSols in the Astronomy category. Once we have it configurable, then we can add it

**Comment by mokun** [comment]
> Could have a social event starting at 800 mSols in the Astronomy category.

But we don't need it every night.

Can we allow the selection of a daily, weekly, bi-weekly, monthly, or yearly group activity?

It would be nice to be like a typical timer app on our phone, or setting up an event on our calendar on that we get to choose the frequency of occurrence precisely.

**Comment by bevans2000** [comment]
> > Could have a social event starting at 800 mSols in the Astronomy category.
> 
> But we don't need it every night.
> 
> Can we allow the selection of a daily, weekly, bi-weekly, monthly, or yearly group activity?
> 
> It would be nice to be like a typical timer app on our phone, or setting up an event on our calendar on that we get to choose the frequency of occurrence precisely.
> 
> 

At the moment the night sky meeting has a repeating frequency so it gets repeated after a number of sols
